rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -919,6 +917,79 @@ rmw_node_t * rmw_create_node(const char * name, const char * namespace_, size_t
   return NULL;
 }
 
+bool rmw_get_security_file_paths(
```

Comment:
This function is not part of the rmw interface and should therefore not have the `rmw_` prefix.